e2e/groups/user-group-invitations.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { expect, Page, test } from '@playwright/test';
+import { initAsDemoUser, initAsUsualUser } from '../helpers/e2e_auth';
+import { apiUrl } from '../helpers/e2e_http';
+
+const groupName = 'E2EGroupInvitationProcess';
+const groupId = '4433009959583369709';
+const demoUserToken = 'demouser';
```

Comment:
You have hardcoded the token in the code?!  🤨 Why have you done differently from the usual user?
If it is a user in dev, we cannot let anyone mess with it. 
Ok, I am going to disable this token. Please use a CI env variable for "secrets" the same I did for the usual user.

Reply:
Fixed